docs/usage.rst

```diff
@@ -73,12 +132,32 @@ Burbach SM, Briney B. Improving antibody language models with native pairing. Pa
 TCR-specific embeddings
 =======================
 
+**Important Note**: Most TCR embedding models listed below are primarily trained on alpha/beta TCRs (TRA/TRB sequences). While AMULETY's unified interface accepts gamma/delta TCRs (TRG/TRD), results may be less reliable for these sequences due to limited training data.
+
 TCR-BERT
 --------
 
-BERT model pre-trained on 88,403 human TCR alpha and beta sequences from VDJdb and PIRD databases. Specialized for T-Cell Receptor analysis.
+BERT model pre-trained on 88,403 human TCR alpha and beta sequences (TRA/TRB) from VDJdb and PIRD databases. Specialized for alpha/beta T-Cell Receptor analysis.
```

Comment:
Can you specify here also the chain, and for all the other models too (see also comment above to move the chain specification to this section).